### ISSUE TITLE
Monitoring threads continue to run after program has ended

### ISSUE THREAD

**Issue by bonzaiferroni** [sirkris/Reddit.NET#74]
First I'll say I'm a hobbyist programmer so I apologize if there is an obvious solution that I don't see. It seems from the example code that the primary way to end monitoring threads is to call the same method that started the thread, to "toggle off". If a thread isn't toggled off, it will continue to execute even after the main thread is completed (at least in the context I'm working in, Unity3D). I continue to see new post messages in the console even after stopping the program.

It would be convenient to have a way to stop all monitoring and any background threads.

**Comment by sirkris** [comment]
I created a new method that allows all of an object's monitoring threads to be terminated with a single call.  Here's a short example:

```c#
comment.Comments.MonitorNew();
comment.Comments.MonitorOld();
comment.Comments.MonitorControversial();

...

comment.Comments.KillAllMonitoringThreads();
```

Merged into 1.3.  You can get it now on the develop branch.  Does this solve your issue?

**Comment by bonzaiferroni** [comment]
Yes it does! Thank you for looking into this.
